tests/interface/geom_input_builtin_array.spv.yaml

```diff
@@ -0,0 +1,208 @@
+%YAML 1.0
+---
+all_type_descriptions:
+  - &td0
+    id: 7
+    op: 23
+    type_name: 
+    struct_member_name: "gl_Position"
+    storage_class: 0 # UniformConstant
+    type_flags: 0x00000108 # VECTOR FLOAT 
+    decoration_flags: 0x00000000 # NONE
+    traits:
+      numeric:
+        scalar: { width: 32, signedness: 0 }
+        vector: { component_count: 4 }
+        matrix: { column_count: 0, row_count: 0, stride: 0 }
+      image: { dim: 0, depth: 0, arrayed: 0, ms: 0, sampled: 0, image_format: 0 } # dim=1D image_format=Unknown
+      array: { dims_count: 0, dims: [], stride: 0 }
+    member_count: 0
+    members:
+  - &td1
+    id: 12
+    op: 28
+    type_name: 
+    struct_member_name: "gl_CullDistance"
+    storage_class: 0 # UniformConstant
+    type_flags: 0x20000008 # ARRAY FLOAT 
+    decoration_flags: 0x00000000 # NONE
```

Comment:
This **should** have a `built-in` label to it, but to be tracked as a future fix (if someone cares enough about geometry)